examples/usecases/accelerate-training-of-large-embedding-tables-by-LazyAdam.ipynb

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    model.fit(train, batch_size=1024, epochs=1)
</pre><p>This should now work if you merge with the <code>main</code> branch. (fixed in  https://github.com/NVIDIA-Merlin/models/pull/759 ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>